CONTRIBUTING.md

```diff
@@ -40,6 +40,21 @@ as well as data-persistence; with two additional submodules.
 - `features`, which includes the high-level test definitions for the Convene
   experience.
 
+### Architecture Overview
```

Comment:
VSC auto inserts table of contents (TOC) links for these new headers. IDK if ya'll wanted it... so I removed it 57b6d6e7dd454361e3b8db708b476475d26649c9. 

Lemme know if you want it back in, and I'll revert the removal and fix prepend nice numbering to the TOC links.

```
1.1.2 Architecture Overview
1.1.3 ERD
```

Reply:
I agree these don't need to be in the ToC. If you set your Markdown All In One Extension's Table-of-Contents depth to `1..3` and use an h4 (`####`), it will not generate sub-sub-sub-headings to the ToC
![Screen Shot 2022-12-16 at 10 33 58 AM](https://user-images.githubusercontent.com/50284/208165693-4a9d9f94-ddb9-4f4e-8beb-9768edf11c7c.png)